docs/endpoints/getGJLevelScores211.md

```diff
@@ -0,0 +1,49 @@
+# getGJLevelScores211.php
+
+Upload and query user levelBoard record.
+
+## Parameters
+
+**gameVersion** - The version of the game
+
+**binaryVersion** - The binary verion of the game
+
+**gdw** -- is geometry dash world
+
+**accountID** - The ID of the account
+
+**gjp** - The password encoded of the account
+
+**levelID** - The ID of the level
+
+**percent** - Percent of the player best
+
+**secret** - Wmfd2893gb7
+
+**type** - 0: friends, 1: top, 2: week
+
+**s1** - attempts + 8354
+
+**s2** - jumps + 3991
+
+**s3** - seconds + 4085
+
+**s4** - 1482 * 2 + (jumps + 3991) * (percentage + 8354) + ((seconds + 4085) ** 2) - 50028039
```

Comment:
not sure if `**` is just a typo for `*` or meant to be `^` but rn it's a bit confusing to read that part

Reply:
Hey, sorry for interrupting. I used `**` since `^` is really XOR operator. If this is quite strange to read, we can probably do `pow(x, 2)` or something.

Reply:
we're writing calculations using mathematical terms, not programming ones since we're not writing code blocks